### DISCUSSION TITLE
Project: Budgeting

### DISCUSSION THREAD

**Discussion by sak007** [sak007/MyDollarBot-BOTGo#30] [General]
Created pull request for Budgeting feature.

Pull Request: https://github.com/sak007/MyDollarBot-BOTGo/pull/23
Project: https://github.com/sak007/MyDollarBot-BOTGo/projects/4

Test and close the issues in the project.
Once testing is done, please approve the pull request.

@zedunaid @sak1997 @wizard-31 @rjain09

**Comment by wizard-31** [comment]
All tests are checked, and build passes without any issues. We're good to go ahead and approve the merge. 
I've approved the PR from my end. 

We're good to proceed.

**Comment by sak1997** [comment]
Tested budgeting and it works fine.

Budget remaining is not showing up after edit; this needs to be fixed once the new updated edit feature has been merged.
